flink-dist/src/main/flink-bin/bin/flink

```diff
@@ -51,5 +62,16 @@ log_setting=(-Dlog.file="$log" -Dlog4j.configuration=file:"$FLINK_CONF_DIR"/log4
 export FLINK_ROOT_DIR
 export FLINK_CONF_DIR
 
+# Get the path of ${FLINK_ROOT_DIR}/opt/flink-table*.jar.
+TABLE_JAR_PATH=`echo "$FLINK_ROOT_DIR"/opt/flink-table*.jar`
+if [[ $(isPython "$@") -eq 0 ]]; then
+	TABLE_JAR_PATH=`echo "$FLINK_ROOT_DIR"/opt/flink-table*.jar`
+	CLASSPATH=`manglePathList "$TABLE_JAR_PATH:$CC_CLASSPATH:$INTERNAL_HADOOP_CLASSPATHS"`
+	# use the -C option to add the flink-table relevant jar to the classpath.
+	arg="-classpath $CLASSPATH org.apache.flink.client.cli.CliFrontend $@ -C file:///$TABLE_JAR_PATH"
```

Comment:
As far as I know, the URL in -C option needs to be accessible by all nodes on the cluster. If it is just a local file path, we must make sure that it is also accessible in other nodes' machines. Perhaps this design only works on local mode?

Reply:
Thank you for finding this error.Because flink currently don't support submitting a list of jars.I choose to put flink-table-*.jar in jarFilePath as a tmpral way.If flink support submitting jars in the future,the relevant realization will be corrected.

Reply:
After considering that jarFile can only contain a jar file that uploaded, I have put flink-jar in the method getAllLibraries of Packagedrogram.Because all jars uploaded will be added in the libs, it looks like a better solution.